pcsx2/Mdec.cpp

```diff
@@ -199,14 +199,14 @@ void mdecWrite1(u32 data) {
 }
 
 u32 mdecRead0(void) {
-	CDR_LOG("mdec0 read %lx", mdec.command);
+	CDVD_LOG("mdec0 read %lx", mdec.command);
 
 	return mdec.command;
 }
 
 u32 mdecRead1(void) {
-#ifdef CDR_LOG
-	CDR_LOG("mdec1 read %lx", mdec.status);
+#ifdef CDVD_LOG
```

Comment:
Additionally, this is a CD system log, converted now to a CDVD log, all in a MDEC (video decoder) file.
It should be its own thing or left out completely.